src/github.com/couchbase/sync_gateway/db/change_cache.go

```diff
@@ -186,12 +186,15 @@ func (c *changeCache) CleanSkippedSequenceQueue() bool {
 		for _, skippedSeq := range c.skippedSeqs {
 			if time.Since(skippedSeq.timeAdded) > c.options.CacheSkippedSeqMaxWait {
 				// Attempt to retrieve the sequence from the view before we remove
-				options := ChangesOptions{Since: SequenceID{Seq: skippedSeq.seq}}
+				options := ChangesOptions{Since: SequenceID{Seq: skippedSeq.seq - 1}}
```

Comment:
Eg, 

```
// get the sequence that happened immediately before the
// skipped sequence, since we will be using that as our "since" parameter
seqBeforeSkippedSeq := skippedSeq.seq - 1
```